challenges/objects-arrays.js

```diff
@@ -46,24 +69,46 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
-/* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
+/* Request 1: Create a new array called universities that contains all the universities in the graduates array.  */
+
+
+
+
+
+
 
-Once you have the new array created, sort the universities alphabetically and log the result. */
+/* Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+graduates.forEach((element)=>{
```

Comment:
You don't need the parenthesis if there is only one parameter in your arrow function :)